### ISSUE TITLE
Level of Effort | Allow Veterans to Submit an SC when the ITF endpoint is down

### ISSUE THREAD

**Issue by saderagsdale** [department-of-veterans-affairs/va.gov-team#82627]
## Value Statement

**_As a_** Veteran attempting to submit a supplemental claim before my Intent to File expires
**_I want to_** complete the online form without being blocked by ITF endpoint outages
**_So that_** I can apply the effective date from my previously submitted ITF to my supplemental claim.

---

## Background Context
Emily Theis is looking for an estimate from on the timeline for implementation - hoping for a matter of days or a week. ([see Slack thread](https://dsva.slack.com/archives/C5AGLBNRK/p1715099714479839))

The VA will begin sending emails to Veteran's who have an ITF on file that is set to expire within 90 days. OCTO wants to allow Veterans to submit a claim even when the ITF endpoint is down. Proposed changes also include displaying an alert on the ITF page.

## Outcome, Success Measure, KPI(S), and Tracking Link
- ITF endpoint outages are not a barrier to form completion
- The new alert doesn't increase form abandonment

## Acceptance Criteria
- [ ] Veterans can submit a claim even when the ITF endpoint is down
- [ ] When the endpoint is down an alert is displayed which instructs Veterans on how to proceed

## Designs and Build Notes
- [Link](https://www.figma.com/file/iRmZuM3vUEYKxeSjTpMdkT/526-ITF-Stopgap?type=design&node-id=10-386&mode=design) to design created by the 526 team

### Enablement team (if needed)
@jstrothman 

## Out of scope
- This ticket is for the LOE only, not code work.

## Open questions (Emily Theis is looking into this now)
- What content is in the email that VA is sending to Veterans?
- Is the design approved for our form or just the 526 form? 

## Tasks
- [ ] Assess the code to determine changes required
- [ ] List any complexities, tech tasks, unit testing needs, and QA requirements in a summary by commenting on this ticket
- [ ] Sade will share the LOE with Emily Theis in [this thread](https://dsva.slack.com/archives/C5AGLBNRK/p1715099714479839)

## Definition of Ready

- [ ] Reviewed by engineering
- [ ] Approved for pickup

## Definition of Done

- [ ] Tech tasks completed
- [ ] Reviewed and approved by product

**Comment by Mottie** [comment]
ITF work on this is being done by the 526 team - [PR #29644](https://github.com/department-of-veterans-affairs/vets-website/pull/29644)

**Comment by Mottie** [comment]
Work has been merged in. It should be available in staging later today, and deployed to production on Monday

**Comment by eileen-coforma** [comment]
@Mottie Anything we can review async here, or during parking lot?

**Comment by eileen-coforma** [comment]
Actually, should this be closed, since work is complete and we don't need an LOE, since Thomas from 526 is doing the work, or should this ticket be renamed?
